corehq/apps/app_manager/models.py

```diff
@@ -1732,6 +1727,11 @@ def case_list_filter(self):
 
     def validate_for_build(self):
         errors = super(Module, self).validate_for_build()
+        if not self.forms and not self.case_list.show:
```

Comment:
Should I also allow building if `referral_list.show` or `task_list.show` are `True`? I don't know exactly how these things are handled, but it looks like they are a little different...